### ISSUE TITLE
Pasting a JSON object

### ISSUE THREAD

**Issue by daryllimyt** [YYsuni/react18-json-view#45]
Hi, this is a great project! Would love to know if it's possible to copy and paste JSON objects to and from the editor?

Thanks!

**Comment by YYsuni** [comment]
Yup, just pasting it with `()` surrounded. eg. `({"k1":123,"k2":"123","k3":false})`. You could try it online: https://jv.yysuni.com/
